### DISCUSSION TITLE
Odin not booting on new Intel Tiger Lake laptop

### DISCUSSION THREAD

**Discussion by vjr** [elementary/triage#127] [Q&A] [answered]
I asked on stackexchange too here https://elementaryos.stackexchange.com/questions/26719/odin-6-0-not-booting-on-intel-tiger-lake-laptop

Trying the latest Odin ISO on a new HP Pavilion 14 laptop with Intel Core i5 1135G7 CPU and Iris Xe iGPU - Tiger Lake platform - does not work - meaning it boots to blank screen - and unresponsive - after a few moments pressing the power key immediately shuts down the laptop indicating some sort of hang.

FYI, the ubuntu 18.04 based Hera boots and installs fine (only wifi works - no sound no trackpad) and also the stock Ubuntu 20.04 with the 5.4 kernel boots into live session, the stock ubuntu 20.10 with kernel 5.8 boots and everything appears to work well.

This current Odin ISO has the latest HWE kernel 5.8 yet it does not boot.

I have tried some kernel boot parameters like nomodeset along with i915.modeset=0 but those didn't work.

I am unable to switch to TTY or anything to look at any logs etc. even if I remove the "quiet splash" options there is just blank screen.

Any suggestions to troubleshoot/fix this issue?

Incidentally, the KDE Neon User edition ISO exhibits the same issue even though it too is based on updated ubuntu 20.04 kernel 5.8.

**Comment by vjr** [comment]
Posting dmesg and lshw output here from stock ubuntu 20.04.2 install...

[dmesg.txt](https://github.com/elementary/os/files/6215033/dmesg.txt)

[lshw.txt](https://github.com/elementary/os/files/6215034/lshw.txt)

**Comment by davidmhewitt** [comment]
Do you have a USB to serial adapter or two? If so, you could connect that and add `console=ttyUSB0,115200` to the kernel parameters. Then connect to that serial port from another PC and see what the kernel is outputting.

Without some kernel output, I don't think there's much we can do.

**Reply by vjr** [reply]
Hey thanks for your comment.

I feel silly now because I was using the Ventoy multi ISO boot tool which seems to be the root cause. I just now used the Rufus tool to write the ISO to a USB stick and it looks like it booted fine!

I will install it and play around a bit and if everything is fine, will close this discussion and report the issue to the Ventoy developer.

Cheers!

**Comment by vjr** [comment, marked as answer]
Follow up to https://github.com/elementary/os/discussions/455#discussioncomment-537763

Looks like Odin daily works well enough after all - if I write the ISO directly to a USB stick and not use the ventoy multi iso boot tool.

It was working with the ventoy tool on my other hardware which is why I guess I thought it must have been the new hardware.

Reported the issue to the Ventoy developer, closing this ticket, sorry for the noise!

**Comment by vjr** [comment]
@davidmhewitt  I've reported this issue to the ventoy developer but I was wondering if there is anything that can be done here to identify whether the issue lies with the ventoy tool or with the odin iso build?

Reason why I ask is because other distro isos work,  including Ubuntu 20.04 and odin works with other hardware with the ventoy tool. I know not necessarily strong evidence pointing anywhere but still?

**Reply by vjr** [reply]
I should mention that it loads up to grub boot screen where the odin logo and countdown happens but subsequent to pressing enter the screen goes blank with unresponsive keyboard...

**Comment by danirabbit** [comment]
Closing as out of date since this was an issue with an older release of Ventoy on an older release of elementary OS
